Resources/Prototypes/_SpaceCats/Electromouse/Actions/electromouse.yml

```
@@ -0,0 +1,129 @@
+
+- type: entity
+  id: ActionElectroMouseElevation
+  name: Возвышение
+  description: Вы достигаете окончательного прогресса своего существования. Ваши способности становятся еще лучше. Вы получаете бесконечную энергию. Постарайтесь выжить, и привести электропитание станции в упадок
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: InstantAction
+    icon: { sprite: _SpaceCats/electromouse.rsi, state: sm_elevation}
+    event: !type:ElectroMouseElevationEvent
+    charges: 1
+    itemIconStyle: BigAction
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)